Source/Applications/SystemCenterNotification/web.config.backup

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this file in the repo?

Reply:
Probably not. Not sure why that's in there